tooling/helpers/file_helper.py

```diff
@@ -2,7 +2,7 @@
 import json
```

Comment:
I know I will always annoy you with it, but I despise f > file and p > path as variable names

Code is meant to be readable (imo) and it doesn't flow if the variable name is only understood after reading the next 3 words

Reply:
I see the problem...however given these functions are all like 5 lines long anyway I would say in this case it makes no difference